Orphanage/README.md

```diff
@@ -0,0 +1,41 @@
+# Orphanage
+DOM Element highlighter tool, updates role attributes of document elements.
```

Comment:
Add a note that points out that this was an experiment and why we thought it important to explore this area. I would also suggest making a comment that this is a work in progress. You may even want to comment on possible future directions for this code, like turning it into a browser extension, etc.